release_notes/41-release-notes.adoc

```diff
@@ -412,6 +413,13 @@ Release date: 11 July 2023
 * Previously, upgrading from RHACS 4.0.2 to RHACS 4.1 using the RHACS Operator would fail because of the deprecated KernelModule collector support in RHACS 4.1.
 The updated image fixes this issue.
 
+[id="resolved-in-version-412_{context}"]
+=== Resolved in version 4.1.2
+
+Release date: 19 July 2023
+
+* {product-title-short} 4.1.2 includes a fix for an issue where an operator-based RHACS installation failed to upgrade from older versions before RHACS 3.71. Because the older versions did not implement PSP auto-sensing on the Kubernetes cluster 1.25 or later.
```

Comment:
```suggestion
* {product-title-short} 4.1.2 includes a fix for an issue where an operator-based RHACS installation failed to upgrade from versions earlier than RHACS 3.71. The failure occurred because the older versions did not implement PSP auto-sensing on the Kubernetes cluster 1.25 or later.
```

The second sentence was a fragment, so I added some wording to turn it into a complete sentence. If I changed the intended meaning with my revision, feel free to revise further.